types/bull-arena/bull-arena-tests.ts

```diff
@@ -1,9 +1,68 @@
 import Arena from 'bull-arena';
 import express from 'express';
 
+import BeeQueue = require('bee-queue');
```

Comment:
FWIW, I declared my imports like

```ts
import * as Bull from "bull";
import * as Bee from "bee-queue";
```

but perhaps yours is more correct? (figuring out the TS/ES6 import syntax for commonjs modules is not my strong suit)

Reply:
I did this because I was getting errors from `npm test` telling me to do it this way :joy: 

I imagine it's something about the cjs require resolving `export default` to `{ default: <default export> }` and the desire to disambiguate those cases

Reply:
If you're not sure whether to choose `import=` vs `import *`, then `import=` is the one you want. =)